utils/utils.py

```diff
@@ -172,12 +172,15 @@ def exec_silent(cmd: str) -> bool:
     return subprocess.call(cmd, shell=True) == 0
 
 
-def exec_custom(cmd: str) -> None:
+def exec_custom(cmd: str, show_errors=False) -> None:
     assert cmd
-    if exec_silent(cmd):
+    if show_errors:
         logging.info('OK')
     else:
-        logging.error('**ERROR**')
+        if exec_silent(cmd):
+            logging.info('OK')
+        else:
+            logging.error('**ERROR**')
```

Comment:
show_errors doesn't affect on logging. Did you test your change?

Reply:
Suggestion:

```python
def exec_custom(cmd: str, show_errors: bool = True) -> None:
    assert cmd

    def handle(result: bool) -> None:
        if not show_errors:
            return

        if result:
            logging.info('OK')
        else:
            logging.error('**ERROR**')

    handle(exec_silent(cmd))
```

Reply:
Done! @knekrasov Modified according to your suggestion.